### ISSUE TITLE
Helipad is listed as an aid to navigation equipment in DCEG 1.5, s. 18.1

### ISSUE THREAD

**Issue by FrankHippmann** [iho-ohi/S-101-Documentation-and-FC#180]
While a **Helipad** may be an equipment for **Building**, etc.,  it may have been inadvertently added to the list of aid to navigation equipment features in section 18.1.

**Comment by TomRichardson6** [comment]
I recall that this was discussed and the change was intentional. 

@JeffWootton do you recall this too? I think this must have been discussed at PT12 or PT13 I recall Mikko having a view here and possibly Klas too.

**Comment by JeffWootton** [comment]
@TomRichardson6 : This was discussed at the PT13 meeting and it was agreed to add `Helipad` as an equipment feature for `Building`, `Landmark` in addition to `OffshorePlatform`, but not to `SpecialPurposeBeacon` (which was the original proposal from (I think) Finland or Norway. Refer to Decision from S-101PT13 Agenda Item 06.1 and Action S-101PT13/16.

The above is what has been reflected in the DCEG. Has it not been applied as specified in the Feature Catalogue?

![image](https://github.com/user-attachments/assets/3dd0b4da-3792-44b9-a181-03218abe6d86)

**Comment by JeffWootton** [comment]
I should also mention that clause 18.1 lists all the possible equipment features, not just those that are relevant to aids to navigation.

**Comment by FrankHippmann** [comment]
> I should also mention that clause 18.1 lists all the possible equipment features, not just those that are relevant to aids to navigation.

My assumption, based on the context of 18.1, was that the 18.1 structure and equipment lists specifically relate to aids to navigation.

At least from the perspective of validation checks, since there are a number of checks relating to specifically aid to navigation structures and equipment, it is important that lists of aid to navigation structures and equipment are defined.

To be clear, I have no problem with **Helipad** being an equipment feature, but there should be a list of aid to navigation equipment features that does not include **Helipad**.

**Comment by MikkoHovi-FI** [comment]
In our use case `Helipad` is an equipment feature of a `Landmark` as an aid to navigation structure.  But it may as well exist on its own. In that regard validation checks like S101_Dev2233 (in its current form) don't make sense for `Helipad`.

**Comment by FrankHippmann** [comment]
The last two comments clarify the initial query. I see that the confusion relates back to S-57, where the master/slave association is used for AtoNs exclusively. The **StructureEquipment** association replaces this in S-101, but **StructureEquipment** is now more general in its application, relating to more than just AtoNs. In line with the proposed guidance update in the DCEG, some S-101 validation checks relating to **StructureEquipment** and AtoNs will need to be reviewed, e.g., **Helipad** could be treated a special case in the validation checks. The review of S-101 validation checks can be conducted once a decision has been made on whether the updated DCEG guidance is sufficient, or whether a change to the FC is required. This issue can be relabeled as such.

**Comment by JeffWootton** [comment]
I may be missing something here, but I am not sure what the issue here is in regard to validation checks. The allowable list of features that can be bound within each named association is defined (for S-101) within Section 25 of the DCEG. For example, `Helipad` can only be associated using the `Structure/Equipment` named association to  `Building`, `Landmark` and `Offshore Platform` (see S-101 DCEG clause 25.16). I am assuming here that this is managed within the Feature Catalogue, and if this is the case why is there a requirement for a validation check?

**Comment by FrankHippmann** [comment]
> I may be missing something here, but I am not sure what the issue here is in regard to validation checks. The allowable list of features that can be bound within each named association is defined (for S-101) within Section 25 of the DCEG. For example, Helipad can only be associated using the Structure/Equipment named association to Building, Landmark and Offshore Platform (see S-101 DCEG clause 25.16). I am assuming here that this is managed within the Feature Catalogue, and if this is the case why is there a requirement for a validation check?

That is correct, but my query was more related to the distinction between structure and equipment and _aid to navigation_ structure and _aid to navigation_ equipment. The S-101 validation checks reference the terms _navigational aid equipment feature_ and _navigational aid structure feature_, and in order for me to implement these checks correctly in software, I need to know what is meant by these terms. An example is check S101_Dev2233.

> For each navigational aid equipment feature which does not have a StructureEquipment feature association to a navigational aid structure feature OR another navigational aid equipment feature. 

As Mikko suggests, this check does not apply to **Helipad**, since **Helipad** may or may not be a navigational aid equipment feature. But check S101_Dev2233 does apply to the other equipment features listed in DCEG 18.1.

Based on the discussion within this issue, I am working with the following definitions:

**Aid to navigational structure **
The list of structure features included defined by DCEG 18.1.

**Aid to navigation equipment **
The list of equipment features included defined by DCEG 18.1, noting that **Helipad** is only considered an aid to navigation equipment if it is an equipment of a **Landmark** and the **Landmark** is a structure for additional aid to navigation equipment features.

If there is agreement that the above definitions are correct and that Jeff's proposed additional paragraph to DCEG 18.1 clarifies this, then this is enough to address my initial query.

**Comment by TomRichardson6** [comment]
Passed to S-101 Validation Check Sub Group to consider inclusion of the proposed definitions within S-158:101 to make this explicit.

**Comment by JeffWootton** [comment]
S-101 DCEG Sub-Group meeting 28/05/25: Keep Issue open until resolution of any possible Validation Checks and the possibility of an additional named Association.

**Comment by JeffWootton** [comment]
I am now of the opinion that the Structure/Equipment named association has been extended (perhaps inadvertently) from the corresponding relationship in S-57 to include relationships that are not related to aids to navigation - the relationship between a helipad on the top of a building as mentioned in an earlier comment may be considered to be a case in point.

Rather than discussing the merit of creating a new association(s) to describe the relationship between features that are not intended as aids to navigation, my suggestion would be to extend the guidance in the DCEG to include all encoding instances where a "structure" may have an "equipment" feature associated with it. This may be done by retaining this guidance at Section 18 (recommended); or moving this guidance to another Section of the DCEG (where - Section 2?). Consideration could also then be given to including other perceived structure/equipment relationships, such as rescue stations and coast guard stations that are not to be encoded as equipment features for the buildings housing them as specified in the last bullet of clause 18.1.
